openlibrary/macros/SocialShare.html

```diff
@@ -3,29 +3,50 @@
 $# :param str page_url: absolute url excluding params
 
 $def embed_iframe(page_url):
-  <iframe frameborder="0" width="165" height="400" src="$(page_url)/-/widget"></iframe>
+<iframe frameborder="0" width="165" height="400" src="$(page_url)/-/widget"></iframe>
```

Comment:
```suggestion
  <iframe frameborder="0" width="165" height="400" src="$(page_url)/-/widget"></iframe>
```
This is the body of the `embed_iframe` template function, and should be indented.  Otherwise, the embed `<iframe>` is visible in the sidebar.